packages/angular_devkit/core/src/json/schema/prompt_spec.ts

```diff
@@ -262,6 +262,45 @@ describe('Prompt Provider', () => {
         .toPromise().then(done, done.fail);
     });
 
+    it('analyzes enums WITH explicit checkbox type', done => {
+      const registry = new CoreSchemaRegistry();
+      const data: any = {};
+
+      registry.usePromptProvider(async definitions => {
+        expect(definitions.length).toBe(1);
+        expect(definitions[0].type).toBe('checkbox');
+        expect(definitions[0].items).toEqual([
+          'one',
+          'two',
+          'three',
+        ]);
+
+        return [{ [definitions[0].id]: 'one' }, { [definitions[0].id]: 'two' } ];
+      });
+
+      registry
+        .compile({
+          properties: {
+            test: {
+              type: 'array',
+              enum: [
+                'one',
+                'two',
+                'three',
+              ],
```

Comment:
This is an invalid schema, something cannot be an array and also be equal to `one`/`two`/`three`. For array definition, the items field should be used. For additional information, please see here: https://json-schema.org/understanding-json-schema/reference/array.html